Sources/Clappr_iOS/Classes/Plugin/Core/MediaControl/PlayButton.swift

```diff
@@ -71,7 +82,18 @@ open class PlayButton: MediaControl.Element {
         show()
         changeToPlayIcon()
     }
+
+    private func onComplete() {
+        show()
+        changeToReplayIcon()
+    }
     
+    private func onWillSeek(_ position: Double) {
```

Comment:
Could we cover this method with more unit tests?

Reply:
Done! Refactored the whole test class.